pkgs/applications/editors/emacs/elisp-packages/manual-packages/acm-terminal/default.nix

```diff
@@ -1,55 +1,34 @@
-{ lib
-, melpaBuild
-, fetchFromGitHub
-, acm
-, popon
-, writeText
-, writeScript
+{
+  lib,
+  acm,
+  fetchFromGitHub,
+  melpaBuild,
+  popon,
+  unstableGitUpdater,
```

Comment:
Just a nit, but for the packages I maintain I prefer to keep these in the same order they're used rather than alphabetical order.